third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -1357,6 +2074,30 @@ static int GetAPIObjectIdCallback(v8::Local<v8::Object> object) {
   return 0;
 }
 
+static void SetDataProperty(v8::Isolate* isolate,
+                            v8::Local<v8::Object> obj,
+                            const char* property,
+                            v8::Local<v8::Value> value) {
+  v8::Local<v8::Context> context = isolate->GetCurrentContext();
+  obj->Set(context, ToV8String(isolate, property), value).Check();
+}
+
+// // Set `obj[property] = impl`
+// // NOTE: this should theoretically work
+// static void SetDataProperty(ScriptWrappable* impl,
+//                             v8::Local<v8::Object> obj,
+//                             const char* property) {
+//   ScriptState* scriptState = ToScriptStateForMainWorld(gLocalFrame);
+//   auto value = impl->WrapV2(scriptState).ToLocalChecked();
+//   auto isolate = scriptState->GetIsolate();
+//   v8::Local<v8::Context> context = isolate->GetCurrentContext();
+//   obj->Set(context, ToV8String(isolate, property), value).Check();
+// }
```

Comment:
This commented out code and the other commented out code below this should be removed.

Reply:
Done.